src/Engine/Spells/CastSpellInfo.cpp

```diff
@@ -2820,28 +2823,38 @@ void CastSpellInfoHelpers::CastSpell() {
                     pParty->pPartyBuffs[PARTY_BUFF_RESIST_EARTH].Apply(
                             pParty->GetPlayingTime() + spell_duration,
                             spell_mastery, amount, 0, 0);
+                    // Spell power for Feather fall and Wizard eye was "spell_level + 5"
```

Comment:
Spell power isn't used for them, right? If this is the case, let's spell this out in this comment.